arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -5831,6 +5835,18 @@
         "modelAsString": false
       }
     },
+    "SigningAlgorithm": {
```

Comment:
Please confirm if this is a definitive list of enum values since modifying it can be a breaking change.
Alternatively you can set ```"modelAsString": true```